docs/layers/accounts/tutorials/terraform-s3-state.md

```diff
@@ -101,8 +114,10 @@ aws --profile acme-gbl-root-admin \
   s3 ls s3://{bucket_name}/vpc/
 ```
 
-:::note
-If a component is mistakenly deployed somewhere and destroyed, a leftover `terraform.tfstate` file will be present on your local filesystem with a small file size so while this is a good way to search for backends, it's not the best way to determine where a component is deployed. Also, the S3 bucket has versioning enabled, ensuring we can always (manually) revert to a previous state if need be.
+:::note If a component is mistakenly deployed somewhere and destroyed, a leftover `terraform.tfstate` file will be
```

Comment:
Why are you removing the newline?

Reply:
It was formatted with prettier. I'll fix this